### ISSUE TITLE
Balance Change Request - Fire Throwers

### ISSUE THREAD

**Issue by Xebenkeck** [UnofficialCrusaderPatch/UnofficialCrusaderPatch#1008]
Hello,

Crusader brought me back again, it's been a while since I've posted here.

But I recently got back into the AIC and AIV editing and I feel a general consensus is that the Arab Fire Throwers are a unit that are pretty underwhelming in both a defensive and offensive standpoint.

I feel this can be broken down to one reason and that is that the Fire Throwers, for some reason are one of the highest attack priorities for other ranged units. I think, don't quote me on this I haven't fully tested it, they are the second highest threat priority for ranged units, only siege equipment seems to be higher in importance for them to fire at.

I feel for defense, yes Fire throwers are great for killing melee units attacking the walls, the problem is that any attacking ranged units almost always kill them off the walls first, so even if you have them for defense they are targeted and killed frequently. The only way I ever get to use them for defense is on my CPU AIV's where I have low walls, but high crenels in front. Towers are suicide for them.

Offensively, they in theory are great, for my Wazir AIC, I have them set as the attunitengage, meaning when a siege starts they should be the units to go to quarries/farms and attack/burn them. Great in theory, but in practice absolutely horrible, they move slowly, and they have short range, and they get targeted first.

I feel the balance change that we should implement with them is change their threat priority/generation.

The game should treat the target that ranged units select as:
-Siege equipment
-S.Bowman/E.Bowman/Xbowman/Horse Archer
-Engineers/Laddermen/Tunnelers
-Slinger/Fire Thrower
-All Melee Units
-Slaves(as a side slaves should be the lowest threat as to give Arab lords a chance at filling in moats) Seriously put two Nizar's against each other and laugh at the ineptitude of them attacking each other.

Not sure where in the game this threat generation is, but if it can be tampered with/altered this would be one of the biggest things 
I would change to help balance the most expensive Arab unit.

Honestly, ask yourself, in the vanilla game, Saladin sends 6 fire throwers in his army, ask yourself have you ever had them get to your walls and throw fire over them and start your castle on fire? I'm pretty sure the answer is no.

Thoughts?

**Comment by gynt** [comment]
Clarification question. Do you want the targeting priority of ranged units to change for all units regardless of the unit owner: Computer player or Human player? Or only for the units of Computer players?

**Comment by Xebenkeck** [comment]
I would say for both human and computer units.

**Comment by jsobeslav** [comment]
I agree that they are very much underutilized in offense (side note: on the other hand, they are IMO better than oil smelter in defense, since they don't rely on any infrastructure).

Is it possible to increase their walking speed and range, and possibly armor versus archers? The goal would be to make them able to survive long enough to get to the walls, and set the castle ablaze from outside of the walls. And on the other hand, to be able to survive a bit longer on the walls when defending.

From my experience with AIs, whenever any AI lord takes them to battle, they arrive to the battlefield way after the rest of the army; often when the battle is already lost - and then they die quickly to the archers.

**Comment by jsobeslav** [comment]
Another possible use for them: could they ignite enemy pitch ditches, and possibly even the spike traps? The goal would be that they could act as some sort of medieval mine sweeper.
